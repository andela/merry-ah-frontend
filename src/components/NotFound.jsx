import React from 'react';
import PropTypes from 'prop-types';
import NavLink from 'react-router-dom/es/NavLink';


const NotFound = ({ location }) => (
    <div className='mt-5 pt-5 container-fluid text-center vh-100'>
      <div>
        <img src={require('../assets/images/404.png')} className='w-50'/>
      </div>

      <h3>Specified route does not exist  <code>{location.pathname}</code></h3>

      <i className='fa fa-arrow-right'/>
      <NavLink to={'/arts'} className='text-uppercase'>View All Arts</NavLink>
      <i className='fa fa-arrow-left'/>
    </div>
);
NotFound.propTypes = {
  location: PropTypes.object.isRequired,
};
export default NotFound;
