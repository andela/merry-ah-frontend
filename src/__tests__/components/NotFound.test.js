/* eslint-disable import/no-unresolved */
import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

let wrapper;
beforeEach(() => {
  const path = { pathname: 'wrongroute' };
  wrapper = shallow(<NotFound location={path}/>);
});
describe('NotFound Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
