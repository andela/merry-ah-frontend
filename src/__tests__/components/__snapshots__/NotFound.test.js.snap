// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component should match snapshot 1`] = `
<div>
  <h3>
    Specified route does not exist  
    <code>
      wrongroute
    </code>
  </h3>
</div>
`;
